### PR TITLE
Adding structseg dataset 

### PR DESCRIPTION
**Pull request by zil317**
Add the dataset from https://structseg2019.grand-challenge.org/
The dataset contains Organ-at-risk segmentation from head & neck CT scans

### CODE REVIEW

**Comment by zil317** [comment]
@googlebot I signed it!

**Comment by zil317** [comment]
@googlebot I fixed it..

**Comment by zil317** [comment]
@googlebot I fixed it.

**Comment by zil317** [comment]
@googlebot I fixed it.

**Comment by Ouwen** [comment]
@zil317 some of your commits are not under your github account which is why the cla is failing

File: tensorflow_datasets/image/structseg.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+"""TODO(structseg): Add a description here."""
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Fix

File: tensorflow_datasets/image/structseg.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import numpy 
+
+# TODO(structseg): BibTeX citation
+_CITATION = """
+"""
+
+# TODO(structseg):
+_DESCRIPTION = """
+"""
+_BASE_URL = """https://structseg2019.grand-challenge.org/"""
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Fix

File: tensorflow_datasets/image/structseg.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+import tensorflow_datasets.public_api as tfds
+import tensorflow as tf
+import os
+import glob
+import nibabel as nib
+import re
+import numpy 
+
+# TODO(structseg): BibTeX citation
+_CITATION = """
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Fix

File: tensorflow_datasets/image/structseg.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+"""
+
+# TODO(structseg):
+_DESCRIPTION = """
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Fix

File: tensorflow_datasets/image/structseg.py [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+"""
+_BASE_URL = """https://structseg2019.grand-challenge.org/"""
+_CITATION = """  XXXX """
+
+
+class Structseg(tfds.core.GeneratorBasedBuilder):
+  """TODO(structseg): Short description of my dataset."""
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Fix

File: tensorflow_datasets/image/structseg.py [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+      for filename in files: 
+          # join the two strings in order to form the full filepath. 
+          filepath = os.path.join(root, filename) 
+          file_paths.append(filepath) 
+
+    # returning all file paths 
+    return file_paths
+
+  def _info(self):
+    # TODO(structseg): Specifies the tfds.core.DatasetInfo object
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Remove todos.

File: tensorflow_datasets/image/structseg.py [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+        # builder.as_dataset.
+        supervised_keys=("image", "label"),
+        # Homepage of the dataset for documentation
+        urls=[_BASE_URL],
+        citation=_CITATION
+    )
+
+  def _split_generators(self, dl_manager):
+    """Returns SplitGenerators."""
+    # TODO(structseg): Downloads the data and defines the splits
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Remove

File: tensorflow_datasets/image/structseg.py [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+            num_shards=1,
+            gen_kwargs={
+                'filepath': path
+            }
+        )
+    ]
+
+  def _generate_examples(self,filepath):
+    """Yields examples."""
+    # TODO(structseg): Yields (key, example) tuples from the dataset
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Remove

File: tensorflow_datasets/image/structseg.py [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+    file_path = filepath  #hard-code, need change
+    
+
```
Comment (Ouwen) [inline tensorflow_datasets/image/structseg.py]: Remove extra lines run the OSS script

**Pull request closed without merging**